app/models/work.rb

```diff
@@ -748,7 +748,10 @@ def chapter_total_display
   # Set the value of word_count to reflect the length of the chapter content
   def set_word_count
     if self.new_record?
-      self.word_count = self.chapters.first.set_word_count
+      self.word_count = 0
```

Comment:
Could we use the same code on both sides of the if statement and get rid of it completely ?